assets/xml/objects/object_skb.xml

```diff
@@ -0,0 +1,64 @@
+<Root>
+    <File Name="object_skb" Segment="6">
+        <!-- Stalchild skeleton -->
+        <Skeleton Name="gStalchildSkel" Type="Normal" LimbType="Standard" Offset="0x41F8"/>
+
+        <!-- Stalchild Limbs -->
+        <Limb Name="gStalchildRootLimb" Type="Standard" Offset="0x40C8"/>
+        <Limb Name="gStalchildWaistLimb" Type="Standard" Offset="0x40D4"/>
+        <Limb Name="gStalchildRoot2Limb" Type="Standard" Offset="0x40E0"/>
+        <Limb Name="gStalchildRightFemurLimb" Type="Standard" Offset="0x40EC"/>
+        <Limb Name="gStalchildRightShinLimb" Type="Standard" Offset="0x40F8"/>
+        <Limb Name="gStalchildRightFootLimb" Type="Standard" Offset="0x4104"/>
+        <Limb Name="gStalchildLeftFemurLimb" Type="Standard" Offset="0x4110"/>
+        <Limb Name="gStalchildLeftShinLimb" Type="Standard" Offset="0x411C"/>
+        <Limb Name="gStalchildLeftFootLimb" Type="Standard" Offset="0x4128"/>
+        <Limb Name="gStalchildRibcageLimb" Type="Standard" Offset="0x4134"/>
+        <Limb Name="gStalchildHeadLimb" Type="Standard" Offset="0x4140"/>
+        <Limb Name="gStalchildJawLimb" Type="Standard" Offset="0x414C"/>
+        <Limb Name="gStalchildRightUpperArmLimb" Type="Standard" Offset="0x4158"/>
```

Comment:
If you're going with femur, you may as well go with humerus here. (Applies elsewhere)

Reply:
Yeah, I didn't know whether to name them anatomically after human bones or not...
thoughts on upper <appendage> and lower <appendage>?

Reply:
I think if you're going to name them after the actual bones, you'd need to name the skull as well, but a number of them are fused: the radius and ulna in the forearm are not separated, for example, and the hand and foot are all one thing. Meanwhile, the spine and rib cage are genuinely separate. So I think it should be bones where possible, limbs when multiple bones are in the same thing. On the other hand, could go the route of naming as much after the limbs as possible. 🤷